build/images/base/Dockerfile

```diff
@@ -13,6 +13,25 @@
 # limitations under the License.
 
 ARG BUILD_TAG
+
+FROM ubuntu:22.04 AS gobgp-binary
+
+ARG BGP_VERSION
```

Comment:
should be `GOBGP_VERSION`

Reply:
Initially I had kept this name BGP_VERSION so that in future any other bgp distribution will also be supported, currently we are specifically going with gobgp so it makes sense name to GOBGP_VERSION